elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -52,13 +62,22 @@ ul.ui-sortable {
   margin-bottom: 0px;
 }
 
+
+ul.submission_table .badge{
+    margin-right: 5px;
+    width: 20px;
+    text-align: center;
+}
+
 ul.ui-sortable li, ul.answer_table li, ul.submission_table li {
   margin-bottom: 5px;
   border: 1px solid grey !important;
   border-radius: 5px !important;
   line-height: 1.2em;
+  padding: 0.4em;
```

Comment:
love the change, but think its a little bit small for me, maybe `0.6em` is good?

Reply:
Yeah, that seems good. The reasoning for 0.4 was it kept the default and code styled blocks about the same width.